heroku/cli
# migrate(W-14179932): pg-v5: Upgrade pg:settings:log-connections

This PR moves `pg:settings:log-connections` to the CLI and migrates to TypeScript

### Test Instructions
1. checkout this branch and run `yarn && yarn build && cd packges/cli`
2. Run `./bin/run pg:settings:log-connections -a YOUR_APP YOUR_POSTGRES`
3. Observe output is the same as the original cli
